apps/src/applab/applab.js

```diff
@@ -231,7 +246,7 @@ function adjustAppSizeStyles(container) {
               // For this scale factor...
               // set the max-height and max-width for the visualization
               childRules[k].style.cssText = "max-height: " +
-                  Applab.appHeight * scale + "px; max-width: " +
```

Comment:
based on the comments above, I think we want these css height "breakpoints" to stay as the original 480px (Applab.appHeight). This should keep scaling behavior the same as before this change, which seems like what we want if the footer is sitting "inside" the 480px boundary. (here and below)